src/users/list_users/v2.rs

```diff
@@ -0,0 +1,107 @@
+//! [GET /_synapse/admin/v2/users/:user_id](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#list-accountshttps://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#query-user-account)
+
+use crate::serde::boolean_as_uint;
+use ruma::{api::ruma_api, UInt};
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+    metadata: {
+        description: "list users endpoint",
+        method: GET,
+        name: "list_users",
+        path: "/_synapse/admin/v2/users",
+        rate_limited: false,
+        authentication: AccessToken,
+    }
+
+    #[derive(Default)]
+    request: {
+        /// Offset in the returned list. Defaults to 0.
+        #[serde(skip_serializing_if = "Option::is_none")]
+        #[ruma_api(query)]
+        pub from: Option<UInt>,
```

Comment:
Since there is a default values, this doesn't need to be an `Option`:

```suggestion
        /// Offset in the returned list.
        ///
        /// Defaults to 0.
        #[serde(default, skip_serializing_if = "ruma::serde::is_default")]
        #[ruma_api(query)]
        pub from: UInt,
```